.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+## Summary
+
+> What is the reason for this Pull-Request? If adding new functionality, please comment:
+> a code snippet showcasing the end-to-end usage of your new methods, and the exact output
+> from this code (print out any returned variables).
+
+
+
+## Impact
+
+> What is the scope of your changes and who/where do you think they will affect?
```

Comment:
Let's make it optional to mention individuals that would be affected by the change.

```suggestion
> 🌐 Areas Affected: Enumerate modules, functions, or documentation impacted by your code.
> 👥 Who’s Affected: Mention who might be impacted (if applicable).
```